### PR TITLE
Discovery single item view

### PR DESCRIPTION
**Pull request by jeremythuff**
WIP On introducing Discovery Views into SAGE.

### CODE REVIEW

File: src/main/java/edu/tamu/sage/controller/handler/DiscoveryViewControllerAdvice.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import edu.tamu.weaver.response.ApiResponse;
+
+@RestController
+@ControllerAdvice
+public class DiscoveryViewControllerAdvice {
+    
+    @ResponseStatus(value = HttpStatus.OK)
+    @ExceptionHandler(DiscoveryContextBuildException.class)
+    public ApiResponse handleDiscoveryContextBuildException(DiscoveryContextBuildException e) {
+        e.printStackTrace();
```
Comment (jcreel) [inline src/main/java/edu/tamu/sage/controller/handler/DiscoveryViewControllerAdvice.java]: if logging, debug

File: src/main/webapp/app/directives/contentEditableDirective.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+sage.directive('contenteditable', ['$sce', function($sce) {
```
Comment (jcreel) [inline src/main/webapp/app/directives/contentEditableDirective.js]: Remove this one

File: src/main/webapp/app/filters/cantaloupeUrlFilter.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+sage.filter('cantaloupeUrl', function () {
+  return function (string) {
+    if (string.indexOf(appConfig.fedoraPath) !== -1) {
+        string = string.split(appConfig.fedoraPath,2)[1];
+    }
+    return appConfig.cantaloupeBaseUrl+btoa("fedora:"+string)+"/info.json";
```
Comment (jcreel) [inline src/main/webapp/app/filters/cantaloupeUrlFilter.js]: Cantaloupe resolver delegate is using UUIDs now

Comment (jeremythuff) [reply src/main/webapp/app/filters/cantaloupeUrlFilter.js]: can this upgrade be handled in an additional feature request?

Comment (jsavell) [reply src/main/webapp/app/filters/cantaloupeUrlFilter.js]: The resolver still supports the old behavior and I think we would need to discuss how the UUID approach could apply to Sage and Cap before trying to implement.

**Pull request closed without merging**